base/system/chkdsk/lang/ro-RO.rc

```diff
@@ -0,0 +1,38 @@
+/*
+ * PROJECT:     Check Disk utility
+ * LICENSE:     See COPYING in the top level directory
+ * PURPOSE:     Romanian resource file
+ * COPYRIGHT:   Copyright 2018 Bișoc George (fraizeraust99 at gmail dot com)
+ */
+ 
+LANGUAGE LANG_ROMANIAN, SUBLANG_NEUTRAL
+
+STRINGTABLE
+BEGIN
+    IDS_USAGE "\n\
+Uz: %s [drive:] [-F] [-V] [-R] [-C] \n\n\
+[drive:]    Specifică driver-ul pentru control.\n\
+-F          Rezolvă erorile discului.\n\
+-V          Vizualizează calea tuturor fișierelor de la disc.\n\
+-R          Găsește sectoarele defecte și recuperează informațiile care pot fi citite.\n\
+-C          Controlează driver-ul doar dacă este necesar.\n\
+\n"
+
+    IDS_PERCENT_COMPL "%d completat.\r"
+    IDS_FILE_SYSTEM "Tipul sistemului de fișier este %s.\n"
+    IDS_ABOUT "\n\
+Chkdskx v1.0.1 de către Mark Russinovich\n\
+Systems Internals - http://www.sysinternals.com\n\
+Adaptarea pentru ReactOS din 1999 de către Emanuele Aliberti\n\n"
+END
+
+STRINGTABLE
+BEGIN
+    IDS_VOLUME_IN_USE "Volumul este folosit și nu poate fi blocat\n"
+    IDS_CHKDSK_FAIL "Operația Chkdsk-ului a eșuat.\n\n"
+    IDS_NO_ENTRY_POINT "Găsirea punctelor de intrare FMIFS este imposibilă.\n\n"
+    IDS_BAD_ARGUMENT "Argument necunoscut: %s\n"
+    IDS_NO_CURRENT_DIR "Obținerea folder-ului respectiv este imposibilă. Cod de eroare: "
```

Comment:
Please translate "directory" as "director". BTW, this term is used in command line context only. Please translate the term "folder", used everywhere else, as "dosar".
I very much insist on this one.